advanced-solidus/images-and-image-processing.md

```diff
@@ -0,0 +1,196 @@
+# Images and Image Processing
+
+Solidus utilizes [MiniMagick](https://github.com/minimagick/minimagick) for [ActiveStorage](https://edgeguides.rubyonrails.org/active_storage_overview.html) and [ImageMagick](https://imagemagick.org/index.php) for [Paperclip](https://github.com/thoughtbot/paperclip#paperclip) to handle the under-the-hood resizing and image processing. After an image is uploaded to a project, it is automatically resized based on when the view requests an image variant "style". The main dictating functions can be found in [adapter](https://github.com/solidusio/solidus/blob/master/core/app/models/concerns/spree/active_storage_adapter/attachment.rb) that controls sizing and the [ActiveStorage](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/image/active_storage_attachment.rb)/[Paperclip](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/image/paperclip_attachment.rb) attachment modules. Solidus is currently in the process of depreciating Paperclip due to ActiveStorage's inclusion with rails.
```

Comment:
Corrected all instances of non-spaced Active Storage

Corrected sequence of steps 

> > After an image is uploaded to a project, it is automatically resized based on when the view requests an image variant "style".
> 

When an image is uploaded to a project, it is automatically resized, processed and saved in your configured storage service. If additional styles are appended after initial setup, it will recognize the image variant does not exist and create then upload the new image variant.